pallets/collator-selection/src/lib.rs

```diff
@@ -292,6 +292,17 @@ pub mod pallet {
 					"invulnerables > T::MaxInvulnerables; you might need to run benchmarks again"
 				);
 			}
+
+			// check if the invulnerables have associated validator keys before they are set
+			for account_id in &new {
```

Comment:
@bkchr There's now a new loop here that's dependent on the number of elements inside of this `new` vector. I see that the weight function already takes in the length of `new` as part of the weight formula, but at the very least, the constant that it needs to multiply by has surely increased due to this loop here.

Reply:
But shouldn't that be taken care of by the benchmarking? Aka we run this function with different number of inputs and that should give us the weight per input size.

Reply:
Yeah, I suppose we just need to run the benchmark again using benchbot and that would suffice.

Reply:
Yeah, but that isn't a requirement for this pr. This needs to be done for a release.

Reply:
Right ok, if it's part of the release process, then we don't need to update the weights in this PR.